pyseir/cli.py

```diff
@@ -7,12 +7,11 @@
 import pandas as pd
 from covidactnow.datapublic import common_init
 
-from structlog_sentry import SentryProcessor
 from multiprocessing import Pool
 from functools import partial
 from libs.datasets import dataset_cache
 from pyseir.inference.initial_conditions_fitter import generate_start_times_for_state
-from pyseir.inference import infer_rt as infer_rt_module
+from pyseir.rt import infer_rt as infer_rt_module
```

Comment:
I would prefer to name this in a way that we don't have to rename the import.

Reply:
Agreed. I add a change that makes the cli function call more consistent (prepend run_), which technically will break backwards compatibility. We can decide before merge if that break is unacceptable, but I think it's only used internally.